Tests/Runtime/Meshes/OutOfPlaceMesh.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using JetBrains.Annotations;
+using UnityEngine;
+
+namespace TestHelper.Monkey.Meshes
+{
+    /// <summary>
+    /// Create a mesh that not contains a Transform position of the GameObject that the component attached to.
+    /// This mesh is able to hit raycast. Without no position annotations, monkey operators cannot operate the mesh
+    /// because a point where the monkey operators operate on is not in the mesh
+    /// </summary>
+    [RequireComponent(typeof(MeshCollider))]
+    [RequireComponent(typeof(MeshFilter))]
+    public class OutOfPlaceMesh : MonoBehaviour
+    {
+        private static readonly Vector3[] s_vertices =
+        {
+            new Vector3(0, 0.5f, 0), new Vector3(-0.5f, 1f, 0), new Vector3(0.5f, 1f, 0),
+        };
+
+        private static readonly int[] s_triangles = { 0, 1, 2 };
+
+        [CanBeNull]
```

Comment:
Do not use `JetBrains.Annotations` in UPM packages.
Because the installation step guide is not yet written.

Reply:
Fixed at [d53bc90](https://github.com/nowsprinting/test-helper.monkey/pull/47/commits/d53bc9039ccdf925362fd29e6d056f45cd5cf4bb).